conferences.md

```diff
@@ -12,21 +12,19 @@ This year's conference is organized by the University of British Columbia.
 
 **Time:** 7am to 5:20pm PST
 
-**Program:** [available here.](https://microeconomics.ca/micro/cetc)
+**Program and instructions:** [available here.](https://microeconomics.ca/micro/cetc)
 
-**Access:** Open to the public. [Please register here.](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg)
+**Access:** open to the public. [Please register here.](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg)
```

Comment:
This should also stay capitalized. It's also much more clear if you separate the registration link from "Access":

```suggestion
**Access:** Open to the public. 

**Registration**: [Please register here](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg).
```